bucket/lazarus.json

```diff
@@ -0,0 +1,50 @@
+{
+    "homepage": "https://www.lazarus-ide.org",
+    "version": "2.0.0",
+    "description": "A Delphi compatible cross-platform IDE for Rapid Application Development.",
+    "license": {
+        "identifier": "GPL-2.0-or-later | FPC-modified-LGPL-2.0-or-later",
+        "url": "https://spdx.org/licenses/GPL-2.0-or-later.html; http://wiki.freepascal.org/FPC_modified_LGPL"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.sourceforge.net/project/lazarus/Lazarus%20Windows%2064%20bits/Lazarus%202.0.0/lazarus-2.0.0-fpc-3.0.4-win64.exe",
+            "hash": "e53f18e15ff94998cda968325214d11326f20f148b47f349230151dbfc11ca70"
+        },
+        "32bit": {
+            "url": "https://downloadssourceforge.net/project/lazarus/Lazarus%20Windows%2032%20bits/Lazarus%202.0.0/lazarus-2.0.0-fpc-3.0.4-win32.exe",
+            "hash": "fb64be2210c3583f3a184a4a3acc2342624582f7d946ab06d4faf64c81e0326d"
+        }
+    },
+    "innosetup": true,
+    "post_install": [
+        "$fpcdir = \"$dir\\fpc\\\" + ([Regex]'fpc-([\\d.]+)').Matches($fname).Groups[1].Value",
+        "& \"$fpcdir\\bin\\x86_64-win64\\fpcmkcfg.exe\" -d \"basepath=$fpcdir\" -o \"$fpcdir\\bin\\x86_64-win64\\fpc.cfg\""
+    ],
+    "shortcuts": [
+        [
+            "lazarus.exe",
+            "Lazarus"
+        ],
+        [
+            "startlazarus.exe",
+            "Lazarus (debug)",
+            "--debug"
+        ]
+    ],
+    "checkver": "lazarus-([0-9.]+)-fpc-(?<fpc>[0-9.]+)-win64.exe",
```

Comment:
```suggestion
    "checkver": "lazarus-([\\d.]+)-fpc-(?<fpc>[\\d.]+)-win64.exe",
```

Reply:
Are these two expressions equivalent?

Reply:
They are the same, but `\d` is almost consistently used in manifests :smile: